texar/utils/beam_search.py

```diff
@@ -154,7 +157,7 @@ def compute_topk_scores_and_seq(
         batch_size: int
         states_to_gather: (possibly nested structure of) decoding states.
 
-    :returns: Tuple of:
+    Returns: Tuple of:
```

Comment:
The reason we use `:returns:` instead of `Returns:` here is because the Sphinx `napoleon` extension incorrectly regards the text between the colons (`Tuple of`) as the return type. The only way to workaround this is to use the raw returns role `:returns:`.